<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_GenerationCage</class>
 <widget class="QDialog" name="Dialog_GenerationCage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cage generation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QListWidget" name="list_maps"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Position attribute :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_positionAttribute">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="group_resolution">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Resolution management</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="check_resolution">
        <property name="text">
         <string>Resolution independence</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>X :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spin_resolution_x">
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Y :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spin_resolution_y">
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Z :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spin_resolution_z">
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_dilate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Dilation management</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QPushButton" name="button_dilaterVoxellisation">
        <property name="text">
         <string>Dilate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_reinitialiserVoxellisation">
        <property name="text">
         <string>Reset</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <spacer name="horizontalSpacer_14">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Level :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_niveauDilatation">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_15">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_generate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Surface generation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="radio_extractionFaces">
          <property name="text">
           <string>Face extraction</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_marchingCube">
          <property name="text">
           <string>Marching cube</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="button_generate">
          <property name="text">
           <string>Generate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_close">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>list_maps</tabstop>
  <tabstop>combo_positionAttribute</tabstop>
  <tabstop>check_resolution</tabstop>
  <tabstop>spin_resolution_x</tabstop>
  <tabstop>spin_resolution_y</tabstop>
  <tabstop>spin_resolution_z</tabstop>
  <tabstop>button_dilaterVoxellisation</tabstop>
  <tabstop>button_reinitialiserVoxellisation</tabstop>
  <tabstop>radio_extractionFaces</tabstop>
  <tabstop>radio_marchingCube</tabstop>
  <tabstop>button_generate</tabstop>
  <tabstop>button_close</tabstop>
  <tabstop>line_niveauDilatation</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_close</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_GenerationCage</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
